views/pages/resources/mental-wellness.pug

```diff
@@ -0,0 +1,27 @@
+doctype html
+html(lang="en")
+    head 
+        title Mental Wellness &#8211; MathSoc
+        include /views/partials/header.pug
+        link(href="/assets/css/pages/wellness.css" rel="stylesheet")
+
+    body
+        include /views/partials/navbar.pug
+        div(class="content")
+            section(class="white")
+                p(class="subheader") #{wellness.subheader}
+                h1 #{wellness.onCampus}
+                ul
+                    each children in wellness.onCampusChildren
+                        if children.link != ""
+                            li #{children.text} 
+                                a(href="" + children.link +"" target="_blank") Click here for the link
```

Comment:
I wonder if there's a more integrated way we could handle the links?  Would we be able to make the link the title of the resource instead of being appended on the end?

Reply:
Hahaha this was something for links I came up with no thought. What do you mean by having it as the title of the resource though?

Reply:
lol no worries, right now it's like:
 - MATES: UW MATES provides peer support to students [Click here for link](https://placekitten.com/1920/1280).

could we instead make it
 - [MATES](https://placekitten.com/1920/1280): UW MATES provides peer support to students.

?

Reply:
i think i came up with a solution? have a look and let me know what you think.